docs/installation.md

```diff
@@ -59,17 +60,79 @@ Click "Start", then just wait for the process to finish.
 
 </div>
 
-<div id="creating-a-usb-on-others" class="slide" markdown="1">
+<div id="creating-a-usb-on-osx" class="slide" markdown="1">
 
-### OS X, Ubuntu, and others
+### OS X
+
+Before creating an install drive, make sure you have all OS X updates installed. To check for available updates, open the Apple Menu and select either "Software Update&#8230;" or "App Store&#8230;" Make sure to install any OS updates in addition to updates labeled with "firmware" and "Boot Camp".
+
+ Next, find out what Mac model you have:
+
+For Macs running OS X Yosemite (10.10) or later, click on the Apple Menu and select "About This Mac". Take note of the year the Mac was build.
+
+![About OS X Yosemite](images/docs/installation/osx_about_yosemite.png)
+
+For Macs running OS X Mavericks (10.9) or earlier, click on the Apple Menu and select "About This Mac" followed by "More Info&#8230;" Take note of the "Model Identifier".
+
+![About OS X](images/docs/installation/osx_about_mac.png)
+
+#### Compatibility
+
+The following Macs are incompatible with elementary OS:
+
+ - PowerPC based Macs
+ - Xserve Macs
+
+It's recommended to create an install USB drive for the following Mac models:
+
+ - MacBook7,1 (Mid 2010) or newer
+ - MacBook Pro7,1 (Mid 2010) or newer
+ - MacBook Air3,2 (Late 2010) or newer
+ - Mac Mini5,1 (Mid 2011) or newer
+ - iMac 13,1 (Late 2012) or newer
+ - Mac Pro5,1 (Mid 2010) or newer
+
+For these Macs, proceed to the instructions on creating an install USB drive. If your Mac is older than the ones shown above, proceed to the instructions on creating an install DVD.
+
+#### Creating an Install USB Drive
+
+Begin with a spare USB flash drive with at least 1 GB of free space.
+
+You'll need a small application called "elementary thumbdrive creator" Download it [here](https://github.com/sdaitzman/elementary-thumbdrive-creator/archive/master.zip), uncompress it, and open the app to launch it. You'll see a window looking like the one below.
```

Comment:
I chose that project, https://github.com/sdaitzman/elementary-thumbdrive-creator/ for the following reasons:

- It uses Automator, which bypasses the OS X Gatekeeper and should be able to run on older versions of OS X
- It uses dd, which, in my experience, is the most reliable way to create a Linux install disk on OS X. Unetbootin never worked reliably for me. 
- It gives clear instructions to the user.

If it's the copywriting (UI text) or name that's an issue, I can fork it and change it. Why do you disagree with recommending it and what do you suggest instead?